src/TestInfrastructure/Framework/ConfigurableSectionController.cs

```diff
@@ -112,7 +113,7 @@ public static ConfigurableSectionController CreateDefault()
             var section = new ConfigurableSectionController
             {
                 ViewModel = new ConfigurableConnectSectionViewModel(),
-                View = new ConfigurableConnectSectionView(),
+                View = Mock.Of<IConnectSectionView>(),
```

Comment:
This was the only place this helper class was used, so I just replaced it with an empty mock.